.circleci/config.yml

```diff
@@ -22,9 +22,10 @@ references:
           PGUSER: postgres
           POSTGRES_DB: correspondence_platform_test
 
-      - image: circleci/postgres:9.5-alpine
+      - image: circleci/postgres:12.3-postgis
```

Comment:
`postgis` seems to be a [packaged collection for geospatial applications](https://postgis.net/install/) 🤔 is it intentional to use this image?